src/components/NewLocationPage/ChartFooter/MetricChartFooter.tsx

```diff
@@ -28,56 +31,115 @@ export interface ShareButtonProps {
   showEmbedButton?: boolean;
 }
 
-const MetricChartFooter: React.FC<{
-  footerText: string;
-  disclaimer?: string;
-  shareButtonProps: ShareButtonProps;
-  aboutModal: AboutModalProps;
-}> = ({ footerText, disclaimer, shareButtonProps, aboutModal }) => {
-  const [openModal, setOpenModal] = useState(false);
+interface DialogProps {
+  open: boolean;
+  closeDialog: () => void;
+  openDialog: () => void;
+  modalContent: MetricModalContent;
+  modalHeader: string;
+}
+
+const ShareButtonBlock: React.FC<ShareButtonProps> = shareButtonProps => {
+  return (
+    <ButtonContainer>
+      <ShareButtons {...shareButtonProps} />
+    </ButtonContainer>
+  );
+};
+
+const MetricModal: React.FC<DialogProps> = ({
+  open,
+  closeDialog,
+  openDialog,
+  modalContent,
+  modalHeader,
+}) => {
   const dialogProps = {
-    open: openModal,
-    closeDialog: () => setOpenModal(false),
-    header: aboutModal.header,
-    links: aboutModal.links,
+    open,
+    closeDialog,
+    header: modalHeader,
+    links: [
+      {
+        cta: 'Learn more',
+        url: modalContent.learnLink,
+        ariaLabel: `Learn more about ${modalHeader}`,
+      },
+    ],
   };
+
   return (
     <>
+      <ModalButton onClick={openDialog}>
+        <AboutText>About this data</AboutText>
+      </ModalButton>
+      <DialogMain {...dialogProps}>
+        <MetricInfoDialogInner modalContent={modalContent} />
+      </DialogMain>
+    </>
+  );
+};
+
+const MetricChartFooter: React.FC<{
+  metric: Metric;
+  projections: Projections;
+  region: Region;
+  stats: any;
+}> = ({ metric, projections, region, stats }) => {
+  const provenance = getSourcesForMetric(
+    projections.primary.annotations,
+    metric,
+  );
+  const overrideDisclaimer = getOverrideDisclaimer(region, metric, provenance);
+  const modalContent = getMetricModalContent(region, metric, provenance);
+  const modalHeader = getMetricNameExtended(metric);
```

Comment:
using extended metric name so the percent vaccinated modal header isnt just 'Vaccinated' (this change also changes Daily new cases --> Daily new cases per 100k population, which doesnt seem like an issue)